src/postDocLoadResourceObserver.js

```diff
@@ -75,6 +76,11 @@ export class PostDocLoadResourceObserver extends InstrumentationBase {
   }
 
   enable() {
+    if (_enabled) {
+      return;
+    }
+    _enabled = true;
```

Comment:
would be correcter to have this on instance (this._enabled) in case someone wants to use this instrumentation twice (not really sure why but kay)

Reply:
Just remove

Reply:
done